workspace-manager-clienttests/src/main/resources/servers/workspace-wsmtest-resiliency.json

```diff
@@ -0,0 +1,24 @@
+{
```

Comment:
Will this get combined with the regular workspace-wsmtest.json when you are done?

Reply:
This one needs to wait. The current `workspace-wsmtest.json` employs a user delegated SA in place of the test runner SA, that was when test runner employs a single SA for testing APIs and reporting test results. Now we want to separate the SAs into 3 types: a) regular IAM Service Account (cluster viewing and results uploading), b) the Kubernetes Service Account (for resiliency test). Test Runner supports both. We need one more Test Runner enhancement to support a user delegated SA as well then we can combine `workspace-wsmtest-resiliency.json` into `workspace-wsmtest.json`. We are not too far away from this.